README.md

```diff
@@ -668,6 +668,108 @@ secret_key = "SCA..."
 
 ```
 
+#### BumpFootprint operation
+
+##### Bump contract
+
+Extends a contract instance lifetime.
+
+**Parameters**
+
+- `contract_address`: Identifier of the contract to be invoked, encoded as `StrKey`.
+- `source_secret_key`: Secret key of the function invoker responsible for signing the transaction.
+- `ledgers_to_bump`: The number of ledgers wanted to extend the contract lifetime.
+
+```elixir
+alias Soroban.Contract
+alias Soroban.RPC.SendTransactionResponse
+
+contract_address = "CD43KXYLGORRXFATEUD3OKOQG4PIKLFL55FRETM3CPHI2WUF2NMFIEUM"
+secret_key = "SDRD4CSRGPWUIPRDS5O3CJBNJME5XVGWNI677MZDD4OD2ZL2R6K5IQ24"
+ledgers_to_bump = 100_000
+
+{:ok, %SendTransactionResponse{hash: hash}} =
+  Contract.bump_contract(contract_address, secret_key, ledgers_to_bump)
+
+{:ok,
+ %Soroban.RPC.SendTransactionResponse{
+   status: "PENDING",
+   hash: "2f6f...",
+   latest_ledger: "279954",
+   latest_ledger_close_time: "1691441432",
+   error_result_xdr: nil
+}}
+
+```
+
+##### Bump contract wasm
+
+Extends the lifetime of a contract's uploaded wasm code.
+
+**Parameters**
+
+- `contract_hash`: Identifier of the contract to be invoked, base 16 encoded.
```

Comment:
Is `contract_hash` the proper name for this? I think it should be `wasm_hash` or `wasm_id`. Please fix if needed in the README and the code